release_notes/Service_Fabric_ReleaseNotes_90.md

```diff
@@ -0,0 +1,153 @@
+| 17. | **Feature** | Made updates to MR UI | **Brief Description**: This update adds new page under each Infrastructure Service section which displays information related to undergoing Infrastructure updates on the Virtual Machine Scale Sets. Users can get information about currently Executing updates, Pending updates and the State of each update right from the Service fabric Explorer.<br>**Impact**: This helps in quickly identifying the undergoing infrastructure updates and understanding overall health of the Cluster.<br>Correlating Infrastructure updates with corresponding repair Tasks , to identify why the updates are stuck.<br>**Documentation Reference**: Read more about it at -<ul><li>[Introduction to the Service Fabric Infrastructure Service](https://docs.microsoft.com/en-us/azure/service-fabric/infrastructure-service-faq)</li><li>[Azure Service Fabric node types and virtual machine scale sets](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-cluster-nodetypes)</li></ul> |
+| 18. | **Feature** | Add Central Secret Store(CSS) to SystemServicesForScale | **Brief Description**: Instantiate the system service Central Secret Store (CSS) with good defaults, omitting the need for additional user-provided configuration settings.<br>**Impact**: Users previously had to specify configuration settings upon activating the Central Secret Store service; failing to specify these settings led to cluster upgrade failures. After this change, enabling the service proceeds with safe defaults for these settings.<br>**Solution**: Add Central Secret Store to the SystemServicesForScale sections of WRPSettings.xml in the WindowsFabric repository, and to Settings.xml in the winfab_rp repository.<br>**Documentation Reference**: Read more about it at -<ul><li>[Enable Central Secrets Service](https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-application-secret-store#enable-central-secrets-service)</li></ul>**Breaking Change**: Attempting to manually set the MinReplicaCount and TargetReplicaCount configuration settings of the Central Secret Store service will be overridden with default values as determined by the runtime for any system service.<br>**Workaround**: [WORKAROUND] |
```

Comment:
What does good defaults mean? Can it be expanded on what we are setting by default?